Dockerfile

```diff
@@ -1,93 +1,131 @@
 ###########################################################
-# Base container with all necessary deps
-# Buster slim python 3.7 base image.
-FROM python:3.7-slim-buster as base
-ENV HTTP_PORT 8080
-
-RUN groupadd -r geoadmin \
-    && useradd -r -s /bin/false -g geoadmin geoadmin \
-    && mkdir -p /logs && chown geoadmin:geoadmin /logs \
-    # install relevent packages
-    && apt-get update \
-    && apt-get install -y binutils libproj-dev gdal-bin \
-    # the following line contains debug tools that can be removed later
+# Container that contains basic configurations used by all other containers
+# It should only contain variables that don't change or change very infrequently
+# so that the cache is not needlessly invalidated
+FROM python:3.9-slim-bullseye as base
```

Comment:
We should stick to the same base image: `3.9-slim-buster` or is there any reason using the bullseye base image ?

Reply:
It's just that bullseye is the newest debian release and buster the older (but still supported) debian release. However, the python images seem to support only the last two debian releases and a new debian release should be published this summer, so thats why I thought that it was a good idea to already upgrade now since I also updated the other dependencies.

Reply:
Ok then that's fine using the last debian version if it is a stable version ?